app/Models/Post.php

```diff
@@ -53,7 +53,22 @@ public function signup()
      */
     public function reactions()
     {
-        return $this->hasMany(Reaction::class);
+        // return $this->morphToMany(Reaction::class, 'postable');
+        return $this->morphMany(Reaction::class, 'postable');
+    }
```

Comment:
@DFurnes we update the model relation last week but after looking at the docs and the examples, I think that we actually don't have a many to many polymorphic relation and just a regular polymorphic relation between post and reactions. Looking at the [polymorphic example](https://laravel.com/docs/5.2/eloquent-relationships#polymorphic-relations), it seems like our reactions are more like the example's "likes" rather than the "tags" in the [many to many polymorphic example](https://laravel.com/docs/5.2/eloquent-relationships#many-to-many-polymorphic-relations) since we don't have different reaction types like a tag will have different tags. 

I wanted to check with you to see if this seems reasonable to you before going down a rabbit hole? The only thing is when I die dump `$post->reactions` in the Post Transformer, I'm still getting this error 
```
local.ERROR: exception 'ErrorException' with message 'array_key_exists(): The first argument should be either a string or an integer' in /home/vagrant/DoSomething.org/rogue/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:2633
``` 
so maybe I'm still a little off with the relations?